src/utils/offers/offerAggregateOptions.js

```
@@ -65,6 +67,14 @@ const offerAggregateOptions = (query, params) => {
     match.languages = { $in: languages }
   }
 
+  if (category) {
+    match.category = mongoose.Types.ObjectId(category)
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)